core/src/validator.rs

```diff
@@ -1254,8 +1257,9 @@ impl Validator {
                 .unwrap()
             };
 
-        let in_wen_restart = config.wen_restart_proto_path.is_some() && !waited_for_supermajority;
-        let wen_restart_repair_slots = if in_wen_restart {
+        let in_wen_restart_phase_one = !waited_for_supermajority
+            && wen_restart_is_in_phase_one(&config.wen_restart_proto_path);
```

Comment:
Does this mean after wen restart is marked done by `wait_for_supermajority` the operator will manually have to delete/move the `wen_restart_proto_path` in order to trigger another wen restart?

Might be good at the start of the validator to immediately exit the validator if the progress path is marked as `Done` already, then i think this check is unnecessary?

Reply:
Nope, the rest is automatic, we are just sharing code path with --wait_for_supermajority so we don't need to write the code again.

We proceed when the progress path is marked as DONE so if some operators are too lazy and don't remove --wen_restart from command line, their validators will still behave normally, so we don't trigger a cluster-wide validator deaths when enough people don't remove --wen_restart after wen restart is done.